### ISSUE TITLE
[data request] CIFAR10-H

### ISSUE THREAD

**Issue by gortizji** [tensorflow/datasets#4603]
* Name of dataset: CIFAR10-H
* URL of dataset: https://github.com/jcpeterson/cifar-10h
* License of dataset: Creative Commons BY-NC-SA 4.0 license
* Short description of dataset and use case(s): A re-labeled version of CIFAR-10's test set with soft-labels coming from real human annotators. For every pair (image, label) in the original CIFAR-10 test set, it provides several additional labels given by real human annotators as well as the average soft-label. The training set is identical to the one of the original dataset. This dataset is useful in the study of label noise and uncertainty in machine learning.

Folks who would also like to see this dataset in `tensorflow/datasets`, please thumbs-up so the developers can know which requests to prioritize.
